src/main/java/net/dv8tion/jda/api/audit/AuditLogKey.java

```diff
@@ -474,6 +475,22 @@
     EMOJI_ROLES_REMOVE("$remove"),
 
 
+    // STICKER
+
+    /**
+     * Change of the {@link net.dv8tion.jda.api.entities.sticker.Sticker#getName() Sticker.getName()} value.
+     *
+     * <p>Expected type: <b>String</b>
+     */
+    STICKER_NAME("name"),
+
+    /**
+     * Change of the {@link Sticker#getFormatType() Sticker.getFormatType()} value.
+     *
+     * <p>Expected type: <b>String</b>
+     */
+    STICKER_FORMAT("format_type"),
```

Comment:
How can this change? What about description and tags?

Reply:
Changed it, only refers to the format typ on creation and it now refers to only description and tags on updates.